DeUrgenta/Features/Menu/Model/MenuOptions.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+enum OptionType {
```

Comment:
Maybe change the name to `MenuOptionType`? I know the enum is declared in this menu model file, but it can also be accessed in other places and the name will be confusing.